### PR TITLE
No results page added

### PR DESCRIPTION
**Pull request by damianmichalowski**
i am added no result page, i made this using vh instead of px, to make this page responsive on all screens
( I based my vh calculations on the sizes of the figma on the screen 1920px)
For now i implemented this to app.js 
` <Route path="movies/noResults" element={<NoResults result="Search" />} />`
but it only for now 
check it out 🔥🔥🔥

### CODE REVIEW

File: src/common/NoResults/styled.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { ReactComponent as noresults } from "../../images/noresults.svg";
+
+export const StyledNoResults = styled(noresults)`
+    width: 668px;
+    height: 508px;
+    align-self: center;
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
+        width: 68vw;
+        height: 54vw;
```
Comment (Marcin-Malek) [inline src/common/NoResults/styled.js:10]: It would be nice if we also had media query for mobile

**Review by Marcin-Malek — APPROVED** [review]
Nice work!

File: src/common/NoResults/styled.js (hunk 1 above)
Comment (JamieShifter) [reply src/common/NoResults/styled.js:10]: I think it looks great just like it is tbh

**Review by JamieShifter — APPROVED** [review]
All good, let's merge it

**Pull request merged**